tests/phpunit/tests/theme/wpTheme.php

```diff
@@ -248,17 +248,17 @@ public function test_network_disable_multiple_themes() {
 	}
 
 	/**
-	 * @dataProvider data_is_block_based
+	 * @dataProvider data_is_block_theme
 	 * @ticket 54460
 	 *
-	 * @covers WP_Theme::is_block_based
+	 * @covers WP_Theme::is_block_theme
 	 *
 	 * @param string $theme_dir Directory of the theme to test.
 	 * @param bool   $expected  Expected result.
 	 */
-	public function test_is_block_based( $theme_dir, $expected ) {
+	public function test_is_block_theme( $theme_dir, $expected ) {
```

Comment:
This test can be simplified to:

```php
	public function test_is_block_theme( $theme_dir, $expected ) {
		$theme  = new WP_Theme( $theme_dir, $this->theme_root );
		$this->assertSame( $expected, $theme->is_block_theme() );
	}
````

Reply:
Good catch.
Refactored in 66e90132749c1f11c6dcb58dedb74a0cbc1cd3fe